integration-cli/docker_cli_start_test.go

```diff
@@ -47,12 +48,12 @@ func (s *DockerSuite) TestStartAttachCorrectExitCode(c *check.C) {
 	dockerCmd(c, "wait", out)
 
 	startOut, exitCode, err := dockerCmdWithError("start", "-a", out)
-	if err != nil && !strings.Contains("exit status 1", fmt.Sprintf("%s", err)) {
-		c.Fatalf("start command failed unexpectedly with error: %v, output: %q", err, startOut)
-	}
-	if exitCode != 1 {
-		c.Fatalf("start -a did not respond with proper exit code: expected 1, got %d", exitCode)
-	}
+	// start command should fail
+	c.Assert(err, checker.NotNil, check.Commentf("startOut: %s", startOut))
+	// start command failed unexpectedly
+	c.Assert(err.Error(), checker.Contains, "exit status 1", check.Commentf("startOut: %s", startOut))
```

Comment:
Could you move the comment in the end of the line or in a `check.Commentf(…)` so we don't loose the "context" when this assertion fails ?